src/gt4py/next/program_processors/runners/dace/transformations/dead_dataflow_elimination.py

```diff
@@ -44,20 +44,24 @@ def gt_eliminate_dead_dataflow(
 
     ret = sdfg.apply_transformations_once_everywhere(
         DeadMapElimination(single_use_data=single_use_data),
-        validate=validate,
+        validate=False,
         validate_all=validate_all,
     )
     ret += gt_dead_memlet_elimination(
         sdfg=sdfg,
         single_use_data=single_use_data,
+        validate=False,
+        validate_all=validate_all,
     )
 
     if run_simplify:
         gtx_transformations.gt_simplify(
             sdfg=sdfg,
-            validate=validate,
+            validate=False,
```

Comment:
```suggestion
            validate=validate,
```
since you are using `elif` below?